docs/_sass/swagger.scss

```diff
@@ -72,28 +74,28 @@ $on-laptop:        800px;
 }
 
 .swagger-method-get {
-    @include swagger-method(#6BBD5B);
+    @include swagger-method(#4fc49d);
 }
 
 .swagger-method-post {
-    @include swagger-method(#248FB2);
+    @include swagger-method(#87adb6);
 }
 
 .swagger-method-put {
-    @include swagger-method(#9B708B);
+    @include swagger-method(#1eaaaf);
 }
 
 .swagger-method-patch {
-    @include swagger-method(#D88541);
+    @include swagger-method(#36bea1);
 }
 
 .swagger-method-delete {
-    @include swagger-method(#E27A7A);
+    @include swagger-method(#87adb6);
```

Comment:
@kaushambisharma I think It would be better if we have a different color for `DELETE `option as in the current one `POST `& `DELETE `have the same styling, what say? 👀
 
@leecalcote @piyushsingariya would love to know your opinion on this

Reply:
Something like this,

![image](https://user-images.githubusercontent.com/23498248/122351759-17581c00-cf6c-11eb-9fb1-c2b51b8ca839.png)

Reply:
Your suggestion falls inline with my feedback above, yes. Pulling colors for the style guide that I linked to above, would be good.

Reply:
okay